lib/screens/config/v2/dup.ex

```diff
@@ -2,25 +2,26 @@ defmodule Screens.Config.V2.Dup do
   @moduledoc false
 
   alias Screens.Config.V2.Header.{CurrentStopId, CurrentStopName}
-  alias Screens.Config.V2.{Departures, EvergreenContentItem}
+  alias Screens.Config.V2.{Alerts, Departures, EvergreenContentItem}
 
   @type t :: %__MODULE__{
           header: CurrentStopId.t() | CurrentStopName.t(),
           evergreen_content: list(EvergreenContentItem.t()),
           primary_departures: Departures.t(),
-          secondary_departures: Departures.t()
+          secondary_departures: Departures.t(),
+          alerts: Alerts.t(),
+          evergreen_content: list(EvergreenContentItem.t())
```

Comment:
merge weirdness

```suggestion
          alerts: Alerts.t()
```

Reply:
Thanks for catching that! I'm sure it's a mistake I made while addressing merge conflicts 😵‍💫